Exiled.API/Features/Lift.cs

```diff
@@ -77,6 +77,11 @@ internal Lift(ElevatorChamber elevator)
         /// </summary>
         public IReadOnlyCollection<ElevatorDoor> Doors => internalDoorsList;
 
+        /// <summary>
+        /// Gets a <see cref="IEnumerable{T}"/> of <see cref="Player"/> in the <see cref="Room"/>.
+        /// </summary>
+        public IEnumerable<Player> Players => Player.List.Where(x => Bounds.Contains(Position));
```

Comment:
Shouldn't this be `Bounds.Contains(x.Position)`? This will get every player otherwise, since the condition will always be `true`.

Reply:
how did neither of you monkeys notice this? i got u